mesonbuild/compilers/compilers.py

```diff
@@ -334,6 +334,9 @@ def is_library(fname):
                 'b_bitcode': coredata.UserBooleanOption('b_bitcode',
                                                         'Generate and embed bitcode (only macOS and iOS)',
                                                         False),
+                'b_pdb': coredata.UserBooleanOption('b_pdb',
+                                                        'Create a PDB file',
```

Comment:
__[Flake8]__

[E127] continuation line over-indented for visual indent

Reply:
Thanks - fixed.